plugins/plugin-chart-table/src/DataTable/DataTable.tsx

```diff
@@ -234,14 +232,16 @@ export default function DataTable<D extends object>({
           </tr>
         )}
       </tbody>
-      {totals && (
+      {shouldRenderFooter && (
         <tfoot>
-          <tr key="totals" className="dt-totals">
-            <td colSpan={totalsHeaderSpan}>{t('Totals')}</td>
-            {totals.map(item => (
-              <td className={item.className}>{item.value}</td>
-            ))}
-          </tr>
+          {footerGroups.map(footerGroup => {
+            const { key: footerGroupKey, ...footerGroupProps } = footerGroup.getHeaderGroupProps();
+            return (
+              <tr key={footerGroupKey || footerGroup.id} {...footerGroupProps}>
+                {footerGroup.headers.map(column => column.render('Footer', { key: column.id }))}
+              </tr>
+            );
+          })}
```

Comment:
Changed to use react-table's built-in footer support to render totals. cc @kgabryje